scss/_carousel.scss

```diff
@@ -158,21 +161,35 @@
   list-style: none;
 
   li {
-    box-sizing: content-box;
+    position: relative;
+    // box-sizing: content-box;
     flex: 0 1 auto;
     width: $carousel-indicator-width;
     height: $carousel-indicator-height;
     margin-right: $carousel-indicator-spacer;
     margin-left: $carousel-indicator-spacer;
     text-indent: -999px;
```

Comment:
Do we really need this? No text is added in the thumbnails anyway. If it needs text, we can work with `aria-labels`?